.github/workflows/coverage.yml

```diff
@@ -1,32 +1,33 @@
-name: Code Coverage
-on: [push, pull_request]
+name: coverage
+
+on:
+  push:
+    branches: 
+      - codecov
+    paths:
+      - tests/CI/*
 
 jobs:
-  run:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
+  build:
+
+    runs-on: ubuntu-latest
+
     steps:
-    - uses: actions/checkout@master
-    - name: Setup Python
-      uses: actions/setup-python@master
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
       with:
-        python-version: 3.7
-    - name: Install Dependencies
+        python-version: '3.x'
+    - name: Install dependencies
       run: |
-        pip install pytest
+        python -m pip install --upgrade pip
         pip install coverage
+        pip install matplotlib
+        pip install numpy
         pip install codecov
-        pip install -r requirements.txt
-    - name: Prepares Code Coverage Report 
-      continue-on-error: true
-      run: |
-        python -m coverage run -m pytest
-    - name: Generate Coverage Report
-      continue-on-error: true
+    - name: Run Tests
       run: |
-        python -m coverage xml
-    - name: Upload to codecov
+        coverage run -m pytest
+    - name: Upload to Codecov
```

Comment:
Why is code coverage being changed on a docs branch?

Reply:
This came from changes in the master branch.